content/brand/icons.md

```diff
@@ -23,9 +23,9 @@ Most of the icons retain the base styling of Feather icons. The 18F version incl
 * Minimum of 2px between elements.
 
 ## Get the icons
-You can access a set of common icons in the [18F Google Slides template]({{ "/brand/templates/" | url }}) (only available to GSA staff), or [download the full set]({{ "/assets/brand/dist/18F_feather.zip" | url }}).
+You can access a set of common icons in the [18F Google Slides template]({{ "/brand/templates/" | url }}) and view the entire icon library [in Figma](https://www.figma.com/design/4REk2fD6r4EmszRHuG1OaF/18F-Feather-Icons?node-id=244-18292) (only available to GSA staff), or [download the full set]({{ "/assets/brand/dist/18F_feather.zip" | url }}).
```

Comment:
Can we indicate that the "full set" download is a zip file, so people aren't surprised when they click on it?

Reply:
Something like:

> or [download the full set (zip)](https://turbo-disco-vgwqj77vjr2x6rv-8080.app.github.dev/assets/brand/dist/18F_feather.zip).

?

Reply:
Maybe "zip file" to make it super obvious to screen readers.